src/test/java/ch/wisv/areafiftylan/integration/UserRestIntegrationTest.java

```diff
@@ -59,13 +60,13 @@
         return profileDTO;
     }
 
-   @Test
+    @Test
     public void testGetAllUsersAsAnonymous() {
         //@formatter:off
         when().
-            get("/users").
```

Comment:
Formatting in this entire class has changed

Reply:
The //@formatter:off statement is apparently an eclipse statement, but the readme recommends using intelliJ. I can revert this change(with some headache), but maybe it is for the better? If this is the case I will remove the comments. Otherwise I will revert this.